Governance/BIDS_Governance_Structure.md

```diff
@@ -0,0 +1,214 @@
+# Brain Imaging Data Structure (BIDS): Governance and Decision Making
+
+Table of Contents
+
+[1. Introduction](#1-Introduction)
+
+[2. Background](#2-Background)
+- [A. Project Summary](#a-Project-Summary)
+- [B. BIDS Mission Statement](#b-BIDS-Mission-Statement)
+
+[3. Leadership Structure](#3-Leadership-Structure)
+
+[4. Governance of the standardization process](#4-Governance-of-the-standardization-process)
+- [A. Principles for open standards development](#a-Principles-for-open-standards-development)
+- [B. Standard decision making process overview](#b-Standard-decision-making-process-overview)
+
+[X. Appendix](#x-Appendix)
```

Comment:
```suggestion
[Appendix](#Appendix)
```

Reply:
erring to keep these indices to purely keep consistent with the original document
imo doesn't hurt

Reply:
okay with me :+1: